docs-chef-io/content/server/v13_2/config_rb_server_optional_settings.md

```diff
@@ -742,11 +742,14 @@ This configuration file has the following settings for `nginx`:
 
 `nginx['ssl_protocols']`
 
-:   The SSL protocol versions that are enabled. For the highest possible
-    security, disable SSL 3.0 and allow only TLS:
+:   The SSL protocol versions that are enabled.For enhanced security set
+    this value to `'TLSv1.2'`. TLS 1.2 is supported on Chef Infra Client
+    releases 10.16.4 and later on Linux/Unix/macOS and 12.8 or later on
+    Windows. If it is necessary to support these older end-of-life
+    Chef Infra Client releases this value can be set to `'TLSv1.1 TLSv1.2'`.
 
     ```ruby
-    nginx['ssl_protocols'] = 'TLSv1 TLSv1.1 TLSv1.2'
+    nginx['ssl_protocols'] = 'TLSv1.2'
     ```
 
     Default value: `TLSv1 TLSv1.1 TLSv1.2`.
```

Comment:
This should be changed to `TLSv1.2` as well

Reply:
These are the 13.2 docs so they still default to the old value